backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/TransferDiskImageCommand.java

```diff
@@ -438,11 +439,18 @@ protected Map<String, Pair<String, String>> getSharedLocks() {
         if (!Guid.isNullOrEmpty(getParameters().getImageId())) {
             List<VM> vms = vmDao.getVmsListForDisk(getDiskImage().getId(), true);
             vms.forEach(vm -> locks.put(vm.getId().toString(),
-                    LockMessagesMatchUtil.makeLockingPair(LockingGroup.VM, EngineMessage.ACTION_TYPE_FAILED_VM_IS_LOCKED)));
+                    LockMessagesMatchUtil.makeLockingPair(LockingGroup.VM, getDiskIsBeingTransferredMessage())));
         }
+
         return locks;
     }
 
+    private String getDiskIsBeingTransferredMessage() {
```

Comment:
1. Looks like there is a kind-of convention to use "LockMessage" suffix for such `new LockMessage` creation methods.
2. What about using this same method (with the new message) in `getExclusiveLocks()`? Currently it doesn't contain the disk name as well.